src/cbmc_starter_kit/template-for-repository/proofs/lib/summarize.py

```diff
@@ -116,4 +147,19 @@ def print_proof_results(out_file):
 
 if __name__ == '__main__':
     args = get_args()
-    print_proof_results(args.run_file)
+    with_aws = args.cloudfront_domain and args.s3_uri
+    without_aws = not args.cloudfront_domain and not args.s3_uri
+    if not with_aws and not without_aws:
+        raise argparse.ArgumentTypeError(
+            "The CloudFront domain and the S3 URI should either be specified "
+            "together or they should not be specified altogether.")
+    if with_aws:
+        CBMC_PROOF_REPORT_HEADER = "Click here to see the CBMC proof report"
```

Comment:
why capitals?

Reply:
Because it is a constant and easier to **not** skip when a future reader goes over this code and wants to update the heading.